### PR TITLE
networkd: add bridge properties

### PR DESCRIPTION
**Pull request by ssahani**
This PR address the RFE  #1172.

[NetDev]  
Name=br-test  
Kind=bridge                                                                                                                                                                              

[Bridge]  
HelloTimeSec=300  
MaxAgeSec=3000  
ForwardDelaySec=288                                                                                                                                                                      

$ brctl showstp br-test
br-test
 bridge id      8000.82dbf39247fe
 designated root    8000.82dbf39247fe
 root port         0            path cost          0
 max age          30.00         bridge max age        30.00
 hello time        3.00         bridge hello time      3.00
 forward delay         2.88         bridge forward delay       2.88
 ageing time         300.00
 hello timer           0.00         tcn timer          0.00
 topology change timer     0.00         gc timer           0.00
 flags

### CODE REVIEW

**Comment by poettering** [comment]
Patch looks OK to me, but I'll let @teg handle this one, as it extends sd-netlink. @teg?

**Comment by ssahani** [comment]
updated the patch .

File: man/systemd.netdev.xml [hunk 1]
```diff
@@ -277,6 +277,43 @@
       </variablelist>
   </refsect1>
 
+    <refsect1>
+    <title>[Bridge] Section Options</title>
+
+      <para>The <literal>Bridge]</literal> section only applies for
```
Comment (torstehu) [inline man/systemd.netdev.xml:283]: should be `<literal>[Bridge]</literal>` here

File: src/network/networkd-netdev.h [hunk 2]
```diff
@@ -141,6 +141,9 @@ struct NetDevVTable {
         /* create netdev, if not done via rtnl */
         int (*create)(NetDev *netdev);
 
+        /* perform aditional configuration after netdev has been createad */
```
Comment (torstehu) [inline src/network/networkd-netdev.h:144]: typo: aditional -> additional

**Comment by poettering** [comment]
@teg please have look!

**Pull request closed without merging**